src/main/java/com/minecolonies/achievements/AbstractAchievement.java

```diff
@@ -0,0 +1,30 @@
+package com.minecolonies.achievements;
+
+import net.minecraft.block.Block;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.stats.Achievement;
+
+/**
+ * Abstraction of the {@link Achievement} class.
+ *
+ * @author Isfirs
+ * @since 0.2
+ */
+public class AbstractAchievement extends Achievement
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this class abstract or rename it, since it matches the regular expression '^Abstract[A-Z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00118)